pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -1,8 +1,28 @@
+# construct an executable file that wraps the actual executable
```

Comment:
Because documentation should always be as close a possible to the actual source code. *Especially* symbol-level documentation.
It won’t be changed until the next time this function is modified, and this is going to trigger a MR anyway.
 
So the documentation in this PR should be reviewed thoroughly and then no mass rebuilds will be necessary (also multiple mass rebuilds can be bundled together).